src/tests/ampDefaultEpic.ts

```diff
@@ -21,22 +20,21 @@ interface AMPEpicResponse {
+    const campaignCode = 'AMP_EPIC_AUGUST2020';
     const currencySymbol = getLocalCurrencySymbol(geolocation);
     return {
         items: [
             {
                 heading: 'Since you’re here...',
                 paragraphs: [
-                    '… we’re asking readers like you to support our open, independent journalism. News is under threat just when we need it the most. Growing numbers of readers are seeking authoritative, fact-based reporting on one of the biggest challenges we have faced in our lifetime. But advertising revenue is plummeting, and many news organizations are facing an existential threat. We need you to help fill the gap.',
-                    'We believe every one of us deserves equal access to quality, independent, trustworthy journalism. So, unlike many others, we made a different choice: to keep Guardian journalism open for all, regardless of where they live or what they can afford to pay. This would not be possible without financial contributions from readers who now support our work from 180 countries around the world.',
-                    'The Guardian’s independence means we can set our own agenda and voice our own opinions. Our journalism is free from commercial and political bias – never influenced by billionaire owners or shareholders.',
-                    'We need your support so we can keep delivering quality journalism that’s open and independent.',
+                    '… we’re asking readers like you to support our open, independent journalism. Millions are flocking to the Guardian for high-quality news every day, and readers in 180 countries around the world now support us financially – this makes our work possible.',
+                    'We believe every one of us deserves equal access to factual information and analysis that has authority and integrity. So, unlike many others, we keep Guardian journalism open for all, regardless of where they live or what they can afford to pay.',
+                    'The Guardian’s independence means we investigate, interrogate and expose the actions of those in power, without fear. Our journalism is free from commercial and political bias – never influenced by billionaire owners or shareholders. This makes us different. We can give a voice to the oppressed and neglected, and stand in solidarity with those who are calling for a fairer future.',
+                    'Every contribution, however big or small, is so valuable for our future.',
                 ],
                 highlightedText: `Support the Guardian from as little as ${currencySymbol}1 – and it only takes a minute. Thank you.`,
                 cta: {
                     text: 'Support the Guardian',
-                    baseUrl: `https://support.theguardian.com/contribute?acquisitionData=%7B%22source%22%3A%22GOOGLE_AMP%22%2C%22componentType%22%3A%22ACQUISITIONS_EPIC%22%2C%22componentId%22%3A%22${campaignCode}%22%2C%22campaignCode%22%3A%22${campaignCode}%22%7D&INTCMP=${campaignCode}`,
```

Comment:
Should this value move to 'url', otherwise won't we lose the tracking links?

Reply:
the client takes care of the full url with tracking links, using `url`, `campaignCode` etc
The `baseUrl` is now unused